ServerCore/Pages/Threads/PuzzleThread.cshtml.cs

```diff
@@ -155,13 +155,15 @@ public async Task<IActionResult> OnGetAsync(int? puzzleId, int? teamId, int? pla
 
         public async Task<IActionResult> OnPostAsync()
         {
-            if (string.IsNullOrWhiteSpace(NewMessage.Text))
+            if (Event.AreAnswersAvailableNow)
```

Comment:
Do we actually need this check? I'm not sure asking for help late is bad per se, it just might not get answered

Reply:
I just didn't want people continuing to add stuff to the server. 
Also, if the answers are available, would you need help?